src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,264 @@
+  </figcaption>
+</figure>
+
+```js/1
+const transformAnimation = modal.animate(openModal, openModalSettings);
+transformAnimation.finished.then(() => { text.animate(fadeIn, fadeInSettings)});
+```
+
+We've chained these animations using `animation.finished.then()` prior to executing the next animation set in the chain. This way, the animations appear in order, and you are even applying effects to different target elements with different options set (such as speed and ease).
+
+Within CSS, this would be cumbersome to recreate, especially when applying unique, yet sequenced animations to multiple elements. You'd have to use a `@keyframe`, sort out the correct timing percentages to place the animations, and use `animation-delay` prior to triggering the animations in the sequence.
+
+### Example: Play, pause, and reverse
+
+What can open, should close! Luckily, since [Chromium 39](https://developers.google.com/web/updates/2014/12/web-animation-playback), the Web Animations API has provided us the ability to play, pause, and reverse our animations.
+
+You can take the above animation, and give it a smooth, reversed animation when clicking the button again using `.reverse()`. This way, you can create a smoother and more contextual interaction for our modal.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-animations/modal-reverse.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    An example of a modal opening and closing upon button click. <a href="https://codepen.io/kevers-google/details/LYpvdNG">See Demo on Codepen </a>
+  </figcaption>
+</figure>
+
+What you can do is create two play-pending animations (`openModal`, and an inline opacity transformation), and then pause one of the animations, delaying it until the other is finished. You can then use promises to wait for each to be finished before playing. Finally, you can check to see if a flag is set, and then reverse each animation.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/waapi-promises?path=script.js&previewSize=0"
+    title="waapi-promises on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+### Example: Dynamic interactions with partial keyframes
+
+<figure class="w-figure">
+<video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-animations/retargetting.mp4">
+  </video>  <figcaption class="w-figcaption">
+    Retargeting example, where a mouse click adjusts the animation to a new location. <a href="https://codepen.io/una/details/BaoveQJ">See Demo on Codepen </a>
+  </figcaption>
+</figure>
+
+```js
+selector.animate([{transform: `translate(${x}px, ${y}px)`}], {duration: 1000, fill: 'forwards'});
+}
```

Comment:
```suggestion
selector.animate([{transform: `translate(${x}px, ${y}px)`}], 
    {duration: 1000, fill: 'forwards'});
```
Removes extra `}` and avoids horizontal overflow 

![image](https://user-images.githubusercontent.com/4713486/83058750-b468af80-a00d-11ea-8ee4-0f54fccf4438.png)